examples/luamtev/hello-world.lua

```diff
@@ -0,0 +1,113 @@
+-- luamtev applications need to be wrapped in a module
+module(...,package.seeall)
+
+-- cache pid/tid
+local pid, tid = mtev.thread_self()
+
+-- initialize RNG
+local s,us = mtev.gettimeofday()
+math.randomseed(us)
+
+function coro_main(coro_id, w)
+  -- Q: Is there any way to get the identity of the currently running coroutine?
+  --    like mtev.thread_self() just for co-routines?
```

Comment:
No. You'd use a lua native one?  Does such a thing exist?

Reply:
There is `coroutine.running()`. Cool.